src/pal/src/misc/errorstrings.cpp

```diff
@@ -38,7 +38,7 @@ ErrorString palErrorStrings[] =
     { ERROR_TOO_MANY_OPEN_FILES, W("The system cannot open the file.\n") },
     { ERROR_ACCESS_DENIED, W("Access is denied.\n") },
     { ERROR_INVALID_HANDLE, W("The handle is invalid.\n") },
-    { ERROR_NOT_ENOUGH_MEMORY, W("Not enough storage is available to process this command.\n") },
+    { ERROR_NOT_ENOUGH_MEMORY, W("Not enough memory is available to process this command.\n") },
```

Comment:
These strings are used on Unix only. The messages are provided by the `FormatMessage` API on Windows. I do not think that this change will do anything to the message in the linked issues. Both of them were on Windows.

Reply:
Suppose its technically correct then :-/
```
ERROR_NOT_ENOUGH_MEMORY
8 (0x8)
Not enough storage is available to process this command.
```